setup/helpers/perf_helper.rb

```diff
@@ -586,4 +586,11 @@ def classify_master_node_via_nc
         'rule' => [ '~', ['fact', 'clientcert'], master_cert_name],
         'classes' => { 'role::puppet_master' => nil } )
   end
+
+  def setup_puppet_metrics_collector_for_foss
```

Comment:
The suite assumes puppet_metrics_collector which wants a String value of a `pe_server_version` fact.